export async function setup() {
  try {
    // to be continued
  } catch (error) {
    console.error("Error in test setup", error);
  } finally {
    // cleanup
  }
}
